src/pavan/Assignment16/FindTheFreq.java

```diff
@@ -0,0 +1,45 @@
+/*
+ Find the frequency of character from a given String array.
+Note: Take String and character from the user using scanner class.
+
+Input : technocredits
+char : e
+output : e -> 2
+ */
+
+package pavan.Assignment16;
+
+import java.util.Scanner;
+
+public class FindTheFreq {
+	int count=0;
```

Comment:
You should write this instance variable with in method as, it’s useful for only that method.  If variable going to use multiple places then it makes senses to define that variable at instance. just trying to explain good coding practices.